etc/kayobe/inventory/group_vars/controllers/network-interfaces

```diff
@@ -1,17 +1,23 @@
+
 ---
 ###############################################################################
 # Network interface definitions for the controller group.
 
-# Overcloud provisioning network IP information.
-br_net_interface: breno3
+# Main bond to 25GbE fabric
+# Note: moved from breno3 1GbE interface
+br_net_interface: brbond0
 br_net_bridge_ports:
-  - eno3
-# TODO: we should have a bond on 25Gb, longer term
-# provision_oc_net_bond_slaves:
+  - bond0
+
+bond0_net_interface: bond0
+bond0_net_bond_slaves:
```

Comment:
Do we need to set any of the bond options here, like bond mode for example?

https://kayobe.readthedocs.io/en/latest/configuration/network.html?highlight=bond

Same comment for other bonds.

Reply:
They are all set in networks.yaml